.github/workflows/monitor-vms.yml

```diff
@@ -28,7 +27,7 @@ jobs:
             | jq -c '[
                 .runners[] 
                 | select(.status == "online")
-                | select(.name | contains("gpu")
```

Comment:
Let's please be more cautious and test thoroughly properly before checking in as Prod code :)